builder/builder_test.go

```diff
@@ -311,15 +311,17 @@ func TestBuildRollupTxs(t *testing.T) {
 	require.Len(t, ethTxs, 2, "Expected two Ethereum transactions: L1 attribute and deposit tx, got %d", len(ethTxs))
 
 	depositTxETH := ethTxs[1]
-	require.NotNil(t, depositTxETH.Value())
+	require.NotNil(t, depositTxETH.Mint())
 	require.NotNil(t, depositTxETH.To(), "Deposit transaction must have a 'to' address")
 
-	cosmAddr := utils.EvmToCosmosAddress(*depositTxETH.To())
+	to, err := gethtypes.NewLondonSigner(depositTxETH.ChainId()).Sender(depositTxETH)
+	require.NoError(t, err)
+	cosmAddr := utils.EvmToCosmosAddress(to)
```

Comment:
nit: might be good to just inline `cosmAddr` and remove the variable since it's only used once

Reply:
We use it twice